### PR TITLE
tailscale: Point to additional config steps

### PR DESCRIPTION
**Pull request by jankatins**
Originally brought up in NAS-132073 and #738

### CODE REVIEW

File: ix-dev/community/tailscale/questions.yaml [hunk 1]
```diff
@@ -64,15 +65,19 @@ questions:
           label: Advertise Exit Node
           description: |
             Advertise exit node</br>
-            Same as `--advertise-exit-node` flag.
+            Same as `--advertise-exit-node` flag.</br>
+            Needs enabled IP forwarding on the host via System > Advanced Settings > Sysctls.</br>
+            See also https://tailscale.com/kb/1019/subnets?tab=linux#enable-ip-forwarding
```
Comment (stavros-k) [inline ix-dev/community/tailscale/questions.yaml:70]:
```suggestion
            Same as `--advertise-exit-node` flag.</br>
            Needs enabled IP forwarding on the host via System > Advanced Settings > Sysctls.</br>
            Please make sure you read and understand the warnings displayed when adding Sysctls</br>
            See also https://tailscale.com/kb/1019/subnets?tab=linux#enable-ip-forwarding
```

File: ix-dev/community/tailscale/questions.yaml [hunk 2]
```diff
@@ -64,15 +65,19 @@ questions:
...
             type: boolean
             default: false
         - variable: advertise_routes
           label: Advertise Routes
           description: |
             Advertise routes</br>
-            Same as `--advertise-routes` flag.
+            Same as `--advertise-routes` flag.</br>
+            Needs enabled IP forwarding on the host via System > Advanced Settings > Sysctls.</br>
+            See also https://tailscale.com/kb/1019/subnets?tab=linux#enable-ip-forwarding
```
Comment (stavros-k) [inline ix-dev/community/tailscale/questions.yaml:80]:
```suggestion
            Please make sure you read and understand the warnings displayed when adding Sysctls</br>
            See also https://tailscale.com/kb/1019/subnets?tab=linux#enable-ip-forwarding
```

**Comment by jankatins** [comment]
Done

**Comment by jankatins** [comment]
I'm currently unsure how my changes could result in the pipeline failure:

```
 Running command [docker run --platform linux/amd64 --quiet --rm -v /home/runner/work/apps/apps:/workspace ghcr.io/truenas/apps_validation:latest apps_render_app render --path /workspace/ix-dev/community/tailscale --values /workspace/ix-dev/community/tailscale/templates/test_values/basic-values.yaml]
========================================+++++========================================
Traceback (most recent call last):
  File "/usr/local/bin/apps_render_app", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/catalog_templating/scripts/render_compose.py", line 47, in main
    render_templates_from_path(args.path, args.values)
  File "/usr/local/lib/python3.11/dist-packages/catalog_templating/scripts/render_compose.py", line 19, in render_templates_from_path
    rendered_data = render_templates(
                    ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/catalog_templating/render.py", line 36, in render_templates
    ).render({'ix_lib': template_libs, 'values': test_values})
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/workspace/ix-dev/community/tailscale/templates/docker-compose.yaml", line 60, in top-level template code
    {% set args = ix_lib.community.tailscale.util.get_args(values.tailscale) %}
^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 485, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'tailscale'
========================================+++++========================================
```

File: ix-dev/community/tailscale/questions.yaml [hunk 3]
```diff
@@ -64,15 +65,21 @@ questions:
...
           schema:
             type: boolean
             default: false
         - variable: advertise_routes
           label: Advertise Routes
           description: |
             Advertise routes</br>
-            Same as `--advertise-routes` flag.
+            Same as `--advertise-routes` flag.</br>
```
Comment (jankatins) [inline ix-dev/community/tailscale/questions.yaml:79]: Unfortunately this whole description blog is missing :-(

image

**Pull request merged**